doc/tutorial/faq.txt

```diff
@@ -0,0 +1,68 @@
+.. _faq:
+
+===========================
+Frequently Asked Questions
+==================================
+How to update a subset of weights?
+==================================
+If you want to update only a subset of a weight matrix (such as
+some rows or some columns) that are used in the forward propogation
+of this iteration, then the cost function should be defined in a way
+that it only depends on the subset of weights that are used in this
+iteration.
+
+For example if you want to learn a lookup table, e.g. used for
+word embeddings, where each row is a vector of weights representing
+the embedding that the model has learned for a word, in each
+iteration only the rows of the matrix should get updated that their
+corresponding words were used in the forward propogation. Here is
```

Comment:
I would say something like: "in each iteration, the only rows that should get updated are those containing embeddings used during the forward propagation"